test/vm-1st-1000.spec.js

```diff
@@ -0,0 +1,71 @@
+'use strict'
+/* eslint-env mocha */
+
+const ethereum = require('../src')
+const expect = require('chai').expect
+const Block = require('ethereumjs-block')
+const BlockHeader = require('ethereumjs-block/header.js')
+// const parallel = require('run-parallel')
+const thousand = require('./data/first-1000-blocks.json')
+const async = require('async')
+const utils = require('ethereumjs-util')
+const rlp = utils.rlp
+
+describe('process 1st 1000 blocks', () => {
+  let eth
+
+  it('spawn a node', () => {
+    eth = new ethereum.Node()
+  })
+
+  it.skip('load pre data to setup vm', (done) => {
+    // eth.vm.setup(testData, done)
+  })
+
+  it('add genesis block', (done) => {
+    const decoded = rlp.decode(thousand[0])
+    console.log(decoded)
+    /*
+    const genesisBlock = new Block()
+
+    genesisBlock.header = new BlockHeader(
+                            testData.genesisBlockHeader
+                          )
+    eth.vm._blockchain.putGenesis(genesisBlock, done)
+    */
+  })
```

Comment:
@wanderer, got the 1st thousand blocks from the blockchain, now I want to run them on the VM. The blocks are rlp encoded on the `first-1000-blocks.json`. 

With regards to https://github.com/ethereumjs/ethereumjs-vm/pull/86#pullrequestreview-162748, what magic do I need to do?

Also, why do we only need to store the BlockHeader for the genesis, would putting the whole block 0 be an issue?